Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -136,6 +136,20 @@ open class KMConversationViewController: ALKConversationViewController {
         sendConversationOpenNotification(channelId: String(describing: channelId))
         setupConversationClosedView()
     }
+    
+    open override func addObserver() {
+        super.addObserver()
+        NotificationCenter.default.addObserver(forName: UIApplication.didBecomeActiveNotification, object: nil, queue: nil) { [weak self] _ in
+            guard let weakSelf = self, weakSelf.viewModel != nil else { return }
+            weakSelf.updateAssigneeDetails()
```

Comment:
If we are updating assignee details when app becomes active, then what about messages and other info? If that's working, then maybe we need to update assignee details from the same place.

Reply:
done. Used observers which is already there `ALKConversationViewController`. Filed an another PR in KMCHATUI: https://github.com/Kommunicate-io/KommunicateChatUI-iOS-SDK/pull/58